libpod/events/journal_unsupported.go

```diff
@@ -0,0 +1,12 @@
+// +build !systemd
+
+package events
+
+import (
+	"fmt"
+)
+
+// newEventJournalD always returns an error if libsystemd not found
+func newEventJournalD(options EventerOptions) (Eventer, error) {
+	return nil, fmt.Errorf("journald logging unsupported")
```

Comment:
im not sure if this is what @rhatdan was getting at but here i would like you to define an actual error in something !system called maybe ErrNoJournalD or something and then return it here.  that way in the future if we need to detect that the error is not generic.  thoughts?

Reply:
Generally agree.

I'd ultimately like to consolidate all libpod errors in a subpackage somewhere (`libpod/error` would shadow Golang's error type, so naming is a problem), but that can definitely happen outside of this PR. For now, a global variable in this package (`ErrNoJournald` or something similar) would be nice, so folks can do an `errors.Cause()` and know that the system doesn't support journald

Reply:
Yes alright, I added a new error `libpod.ErrNoJournaldLogging`.